monitoring-service/src/main/java/org/terracotta/management/service/monitoring/MonitoringServiceProvider.java

```diff
@@ -42,7 +46,8 @@ public boolean initialize(ServiceProviderConfiguration configuration) {
     MonitoringServiceConfiguration config = configuration instanceof MonitoringServiceConfiguration ?
         (MonitoringServiceConfiguration) configuration :
         new MonitoringServiceConfiguration();
-    this.monitoringService = new MonitoringService(config);
+    SequenceGenerator generator = new BoundaryFlakeSequenceGenerator(TimeSource.BEST, NodeIdSource.BEST);
```

Comment:
hum, I guess this is ok to "hardwire" with BEST, since best tries to load VM props.
I would not put this in the tc config, it's really not a standard configuration, rather a very specialized one
It's fine the way it is, or using the default constructor